packages/botonic-plugin-contentful/src/nlp/stemmers/stemmer-uk.ts

```diff
@@ -0,0 +1,88 @@
+import Stemmer from '@nlpjs/core/src/stemmer'
```

Comment:
in the future, when importing an external file to slightly modify it, please create a commit with the exact external version , and one commit with the change.
In this case, it's enough if you add a comment in the stemmer specifying what was the original line